README.md

```diff
@@ -1,134 +1,142 @@
 
-| algorithm | description | classification/regression | protected attributes | supported fairness definitions |
+- _Quality-of-service harms._ Quality of service refers to whether a system works as well for one person as it does for another, even if no opportunities, resources, or information are extended or withheld.
+
+We follow the approach known as **group fairness**, which asks: _Which groups of individuals are at risk for experiencing harms?_ The relevant groups need to be specified by the data scientist and are application specific.
+
+Group fairness is formalized by a set of constraints, which require that some aspect (or aspects) of the AI system's behavior be comparable across the groups. The fairlearn package enables assessment and mitigation of unfairness under several common definitions.
+To learn more about our definitions of fairness, please visit our [terminology page](TERMINOLOGY.md#fairness-of-ai-systems).
+
+>_Note_:
+> Fairness is fundamentally a sociotechnical challenge. Many aspects of fairness, such as justice and due process, are not captured by quantitative fairness metrics. Since there are many quantitative fairness metrics, which cannot be all simultaneously satisfied, our goal is to enable humans to try out different mitigation strategies and then make suitable trade-offs depending on the context.
+
+## Overview of fairlearn
+
+The `fairlearn` package contains the following algorithms for mitigating unfairness in binary classification and regression:
+
+| algorithm | description | classification/regression | sensitive features | supported fairness definitions |
 | --- | --- | --- | --- | --- |
-| `fairlearn.` `reductions.` `ExponentiatedGradient` | Black-box approach to fair classification described in the paper [A Reductions Approach to Fair Classification](https://arxiv.org/abs/1803.02453)| binary classification | categorical | DP, EO |
-| `fairlearn.` `reductions.` `GridSearch` | Black-box approach described in Section 3.4 of the paper [A Reductions Approach to Fair Classification](https://arxiv.org/abs/1803.02453)| binary classification | binary | DP |
-| `fairlearn.` `reductions.` `GridSearch` | Grid search for regression | regression | binary | BGL |
-| `fairlearn.` `post_processing.` `ThresholdOptimizer` | Post-processing algorithm based on the paper [Equality of Opportunity in Supervised Learning](https://arxiv.org/pdf/1610.02413.pdf)| binary classification | categorical | DP, EO |
+| `fairlearn.` `reductions.` `ExponentiatedGradient` | Black-box approach to fair classification described in [A Reductions Approach to Fair Classification](https://arxiv.org/abs/1803.02453)| binary classification | categorical | DP, EO |
+| `fairlearn.` `reductions.` `GridSearch` | Black-box approach described in Section 3.4 of [A Reductions Approach to Fair Classification](https://arxiv.org/abs/1803.02453)| binary classification | binary | DP |
```

Comment:
`GridSearch` can now use the Equalized Odds moment as well.

Reply:
added